spec/lib/msf/ui/console/command_dispatcher/db_spec.rb

```diff
@@ -65,6 +65,73 @@
   it { is_expected.to respond_to :set_rhosts_from_addrs }
 
   describe "#cmd_creds" do
+
+    describe "-u" do
+      let(:username)            { "thisuser" }
+      let(:password)            { "thispass" }
+      let(:nomatch_username)    { "thatuser" }
+      let(:nomatch_password)    { "thatpass" }
+      let(:blank_username)      { "" }
+      let(:blank_password)      { "" }
```

Comment:
These two appear unused.  I imagine you intended to use at least ```blank_username``` on line ~98 rather than ""